__main__.py

```diff
@@ -17,15 +17,39 @@
 """
 
 import logging
+import click
 from pathlib import Path
-
+from peekingduck.cli import cli
+from peekingduck.utils.logger import LoggerSetup
 import peekingduck.runner as pkd
 
-if __name__ == "__main__":
-    run_path = Path.cwd() / "PeekingDuck" / "run_config.yml"
 
+@cli.command()
+@click.option(
+    "--config_path",
+    default=None,
+    type=click.Path(),
+    help=(
+        "List of nodes to run. None assumes run_config.yml at current working directory"
+    ),
+)
+@click.option(
+    "--log_level",
+    default="info",
+    help=("""Modify log level {"error", "warning", "info", "debug"}"""),
```

Comment:
Parentheses may not be required. And slight inconsistency to the actual range of acceptable values? `CRITICAL` is an allowed value in `log_level_settings`

Reply:
Originally, I left out `CRITICAL` as it is seldom used. But added it back now to support the full range.
Also removed parentheses.